### DISCUSSION TITLE
Chibi Monika easter egg

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#7773] [QA - tech support + general]
I saw this Easter egg, but I didn't click on it, and I want to know what happens if I click on it? what is the chance of this easter egg appearing?

**Comment by ghost** [comment]
> I saw this Easter egg, but I didn't click on it, and I want to know what happens if I click on it? what is the chance of this easter egg appearing?

what is the easter egg? when and where is it i wanna know!

**Comment by Nikv2005** [comment]
> > I saw this Easter egg, but I didn't click on it, and I want to know what happens if I click on it? what is the chance of this easter egg appearing?
> 
> what is the easter egg? when and where is it i wanna know!

When Monika show floating islands there is the chance that on middle Island appears chibi version of Monika

**Comment by ThePotatoGuy** [comment]
I will confirm that it exists and it can appear again, but I won't reveal the chance values or potential interactions with it.

**Reply by XxPower-Girl-89xX** [reply]
Yes.. That Happened To Me To.. All She Said Was.. -I Think- "What? How Did That Get There. Give Me A Second..." (Deletes Chibi) "There" And Then It Was Back To Normal
